frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -25,6 +30,26 @@ import { commitRequest, commitDraftRequest } from '../../store/controller/action
 import store from '../../store';
 import { DASHBOARD_ROUTE } from '../navigation/constants';
 
+/**
+ * @typedef Data
```

Comment:
What does a "Data" represent? Is it a field of `ParametersRequest` and/or `AlarmLimitsRequest`, e.g. fio2 or spo2?

Reply:
No, it is an internal type defined as interface in MultiStepWizard itself. I does has props that take `stateKey` as input, which can be (fio2, spo2,...)